bcda/swaggerui/swagger.json

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
Do we really want to commit this to the repo?  I was thinking the documentation would be generated dynamically?

Reply:
It is generated dynamically.  It will be created fresh on docker-compose up. I included it in case anyone wanted to look at it offline or locally.  I can add to .gitignore if you think that would be better.